### ISSUE TITLE
Bug: In-App Messages loading indicator

### ISSUE THREAD

**Issue by c3bryant** [UCSD/campus-mobile#639]
Time category https://its-pro.ucsd.edu/browse/MA-78

Improve loading on the Notifications via a clear loading indicator and accompanying message to the user

User-facing loading messages:
```javascript
if (isLoading) {
	myMessagesStatusText = 'Loading your notifications, please wait.'
} else if (this.props.myMessagesError) {
	myMessagesStatusText = 'There was a problem fetching your messages.\n\n' +
						   'Please try again soon.'
} else if (messages && messages.length === 0) {
	myMessagesStatusText = 'You have no notifications.\nYou may be opted out of all topics.\n\n' +
						   'Notifications to specific topics can be turned on in User Profile.'
}
```

**Comment by g9singh** [comment]
merged into experimental-v2: 45bd03e9c0c1e3a55d660c032fafd37ba03aa24c

**Comment by c3bryant** [comment]
Verified. Merged into `experimental`
